mysql-test/mroonga/storage/column/int/t/with_index_null_value.test

```diff
@@ -0,0 +1,42 @@
+# Copyright(C) 2024 Takuya Kodama <otegami@clear-code.com>
```

Comment:
We should use this format: https://github.com/mroonga/mroonga/blob/56052c854466420ba4df39847561bbccd5993838/COPYING#L474

```suggestion
# Copyright (C) 2024  Takuya Kodama <otegami@clear-code.com>
```

(Old tests don't use the format...)

The following mode line is better for Emacs users like me:

```suggestion
# -*- mode: sql; sql-product: mysql -*-
#
# Copyright (C) 2024  Takuya Kodama <otegami@clear-code.com>
```

See also: https://github.com/mroonga/mroonga/blob/56052c854466420ba4df39847561bbccd5993838/mysql-test/mroonga/storage/index/desc/t/mariadb.test#L1

Reply:
fix: [c3b9abc](https://github.com/mroonga/mroonga/pull/790/commits/c3b9abc9cfcc561ebda54e4a6cecf326a0409af8)